src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -125,10 +125,28 @@
     "drop": PREFIX + 239,
     "decision": PREFIX + 240,
     "keep": PREFIX + 241,
+    # Spans
+    "span.op": PREFIX + 242,
+    "span.module": PREFIX + 243,
+    "span.action": PREFIX + 244,
+    "span.system": PREFIX + 245,
+    "span.status": PREFIX + 246,
     # GENERAL/MISC (don't have a category)
     "": PREFIX + 1000,
 }
-SHARED_STRINGS = {**SESSION_METRIC_NAMES, **TRANSACTION_METRICS_NAMES, **SHARED_TAG_STRINGS}
+
+# 400-499
+SPAN_METRICS_NAMES = {
+    "s:transactions/span.user@none": PREFIX + 400,
+    "s:transactions/span.duration@millisecond": PREFIX + 401,
```

Comment:
Why is span duration a set? Shouldn't it be a distribution?

Reply:
Yes, good catch! A typo copy-pasting I guess.